plugins/grafana-sankey-plugin/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "antreaflowvisibility-grafana-sankey-plugin",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "grafana-toolkit plugin:build",
+    "test": "grafana-toolkit plugin:test",
+    "dev": "grafana-toolkit plugin:dev",
+    "watch": "grafana-toolkit plugin:dev --watch",
+    "sign": "grafana-toolkit plugin:sign",
+    "start": "yarn watch"
+  },
+  "author": "antreaflowvisibility",
```

Comment:
nit: why not antrea?

Reply:
The author name is determined by Grafana cloud account userID. That account was registered by our antrea-flow-visibility email address. I guess that's the reason Kai named it with `antreaflowvisibility` instead of `antrea`.